### ISSUE TITLE
This should have a url of a badger pic

### ISSUE THREAD

**Issue by jinleileiking** [Ullaakut/astronomer#5]

**Comment by Ullaakut** [comment]
?

**Comment by Ullaakut** [comment]
🤷‍♂

**Comment by jinleileiking** [comment]
I mean . I can got a pic of real stargazers from a url

**Comment by Ullaakut** [comment]
Sorry I'm really not sure I understand, could you rephrase your request?

**Comment by jinleileiking** [comment]
This project is to get how many real people started at a project right？

**Comment by jinleileiking** [comment]
Then a url link to show these data is very useful

**Comment by jinleileiking** [comment]
Yeah I see a astro rating badge

**Comment by Ullaakut** [comment]
Currently there are GitHub badges you can get that give you the overall score of a repository, and in the 1.2 version there will be a webapp showing the detailed reports for each repo.

**Comment by jinleileiking** [comment]
Thats what I said
